README.md

```diff
@@ -26,6 +26,7 @@ The conference dataset hosted in this repository tracks the following informatio
 |------------|-----------|-----------|--------------|---------------------|----------------------|  
 | [Functional Scala](https://functionalscala.com) | Ziverge | Yes | Yes | Partial | Never |
 | [LambdaConf](https://lambdaconf.us) | Ziverge | Yes | Yes | Partial | Never |
+| [Scala.IO](https://scala.io) | Scala.IO | "I would prefer not to" | Yes | "Full, to the capacity of the funds" | Never |
```

Comment:
```suggestion
| [Scala.IO](https://scala.io) | Scala.IO | Yes | Yes | Partial | Never |
```

In order to standardize the data in this table, I would suggest to change:

 - "I would prefer not to" to "Yes". Why? Because the table only tracks if a conferrence is *willing* to sign the SEA **if requested to do so by a speaker invited to speak**. It does not track a conference's preferences: some conferences may prefer to sign no agreement, but they may be willing to sign an agreement if explicitly requested to do so. In such a case, the column should be "Yes".
 - "Full, to the capacity of the funds" to "Partial". Why? Because it implies that the conference cannot guarantee full reimbursement, which is correct. LambdaConf is in a similar situation where it has historically tried very hard to cover speaker lodging and travel but due to financial limitations often cannot fully cover everything for everyone.

How does that sound?

Reply:
- My position is : I will sign it if asked, I am not willing to deal with this kind of request. (don't have time, energy, didn't check the document and the implication). I will probably discourage the speaker to it if asked to (the organization of community conference is already taking a toll on our lives each year, in a very similar way as open-source contribution).
- For the past 10 years, we did it fully, and there are some funds left if needed. The risk is low.

I will think about it, it's not urgent for us to state on the matter, the conference will be announced in a couple of weeks. 

If you have mores columns to be more precise about the nature of the conference to clarify the speaker-conference relationship, I take it. For example, We are not paid for the time we spend on it. We don't even have free tickets to spread! (maybe we should 😅)